patches/minecraft/net/minecraft/world/level/block/entity/BrewingStandBlockEntity.java.patch

```diff
@@ -1,5 +1,16 @@
 --- a/net/minecraft/world/level/block/entity/BrewingStandBlockEntity.java
 +++ b/net/minecraft/world/level/block/entity/BrewingStandBlockEntity.java
+@@ -91,8 +_,8 @@
+ 
+    public static void m_155285_(Level p_155286_, BlockPos p_155287_, BlockState p_155288_, BrewingStandBlockEntity p_155289_) {
+       ItemStack itemstack = p_155289_.f_58975_.get(4);
+-      if (p_155289_.f_58979_ <= 0 && itemstack.m_150930_(Items.f_42593_)) {
+-         p_155289_.f_58979_ = 20;
++      if (p_155289_.f_58979_ <= 0 && (itemstack.m_150930_(Items.f_42593_) || net.minecraftforge.common.BrewingFuelRegistry.get(itemstack.m_41720_()) != 0)) {
++         p_155289_.f_58979_ = net.minecraftforge.common.BrewingFuelRegistry.get(itemstack.m_41720_());
```

Comment:
You should move `BrewingFuelRegistry.get()` to its own variable. This will avoid calling it twice, along with the parameters which also include method calls.

Reply:
I've fixed that. I'm not sure why I didn't have it do that originally.